system-test/src/systemTest/resources/log4j.xml

```diff
@@ -10,13 +10,13 @@
     </appender>
 
     <category name="com.containersol.minimesos">
-        <level value="DEBUG"/>
+        <level value="ERROR"/>
     </category>
     <category name="org.apache.mesos.elasticsearch">
-        <level value="DEBUG"/>
+        <level value="ERROR"/>
```

Comment:
I think ERROR is too high. There is no way (AFAIK) for the user to change the log level. If a user has a problem, the logs won't be detailed enough to fix. Set to INFO.